packages/terra-action-header/src/ActionHeader.module.scss

```diff
@@ -2,11 +2,17 @@
 
 :local {
   .left-buttons > * {
-    margin-right: var(--terra-action-header-margin-right, 1.429rem);
```

Comment:
The internal margins/padding of the action-header-container paired with buttons and labels needs some adjustments. Inspecting and will provide feedback.

Reply:
@neilpfeiffer, any update on it?

Reply:
Changes in [ActionHeaderContainer.module.scss](https://github.com/cerner/terra-core/blob/d72d466699b8e5403105cb9bac60e7bf85d8204d/packages/terra-action-header/src/ActionHeaderContainer.module.scss):

```
.flex-header {
    ...
    background-color: var(--terra-action-header-background-color, #f9f9f9);
    border-bottom: var(--terra-action-header-border-bottom, 1px solid #d3d4d5);
    ...
    min-height: var(--terra-action-header-min-height, 2.92857rem);
    padding: var(--terra-action-header-padding, 0.35714rem 0.35714rem 0.35714rem 0);
    ...

    .title {
        font-size: var(--terra-action-header-font-size, 1.10714rem);
        font-weight: var(--terra-action-header-font-weight, 500);
        ...
        padding: var(--terra-action-header-title-padding, 0.28571rem 0.71428rem 0 0.71428rem);
        padding-top: var(--terra-action-header-title-padding-top, 0.28571rem);
        ...
  }
}
```

Reply:
Here: https://github.com/cerner/terra-core/pull/2714/commits/1c4035f5a33f0f8cb5a052ba96d208093538522b